### DISCUSSION TITLE
mainsail doesnt display speed

### DISCUSSION THREAD

**Discussion by hunter-goller** [Desuuuu/klipper#104] [General]
![Screenshot_20211221-153242_Chrome](https://user-images.githubusercontent.com/65684802/146994006-f668428a-bbe1-4383-a2e8-f3b251b28786.jpg)

Mainsail diesnt display the flow or speed.

**Comment by Desuuuu** [comment]
I don't think that's related to my changes.

It might be because the version of Klipper was too old. I just updated the two branches, so you might want to update to the latest version.
